### PR TITLE
docs(dbt-cloud): add instructions to cache dbt Cloud compilation

### PR DESCRIPTION
**Pull request by rexledesma**
### Summary & Motivation
As the title.

### How I Tested These Changes
local

### CODE REVIEW

**Comment by rexledesma** [comment]
Current dependencies on/for this PR:
* master
  * **PR #11668** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11668" target="_blank">Graphite</a> 
    * **PR #11673** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11673" target="_blank">Graphite</a> 
      * **PR #11690** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11690" target="_blank">Graphite</a> 
        * **PR #11691** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11691" target="_blank">Graphite</a> 
          * **PR #11744** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11744" target="_blank">Graphite</a> 
            * **PR #11792** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11792" target="_blank">Graphite</a> 
              * **PR #11793** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11793" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/11793?utm_source=stack-comment).

**Comment by rexledesma** [comment]
Current dependencies on/for this PR:
* master
  * **PR #11668** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11668" target="_blank">Graphite</a> 
    * **PR #11673** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11673" target="_blank">Graphite</a> 
      * **PR #11690** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11690" target="_blank">Graphite</a> 
        * **PR #11691** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11691" target="_blank">Graphite</a> 
          * **PR #11744** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11744" target="_blank">Graphite</a> 
            * **PR #11792** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11792" target="_blank">Graphite</a> 
              * **PR #11793** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11793" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/11793?utm_source=stack-comment).

File: docs/content/integrations/dbt-cloud.mdx [hunk 1]
```diff
@@ -130,6 +130,21 @@ defs = Definitions(
 )
 ```
 
+## Step 4: Set up a GitHub action to cache your dbt Cloud job's compilation
+
+<Note>This step is optional.</Note>
+
+In Step 2, your dbt project is compiled when your Dagster definitions are first loaded. Because this compilation step occurs at runtime, you may encounter high latency when you load your code locations as you wait for the compilation to complete. Instead of compiling at runtime, a cached compilation can be used. A reference to this cached compilation can be created using a GitHub action on your dbt Cloud project.
```
Comment (OwenKephart) [inline docs/content/integrations/dbt-cloud.mdx:137]: "at runtime" isn't the most precise description here -- maybe more like "when your repository is loaded"

**Review by OwenKephart — APPROVED** [review]
note comment, otherwise looks good

**Pull request merged**